src/main/kotlin/io/zerobase/smarttracing/models/Models.kt

```diff
@@ -12,3 +12,5 @@ inline class OrganizationId(override val value: String): Id
 inline class ScannableId(override val value: String): Id
 
 data class IdWrapper(val id: Id)
+
+data class SiteResponse(val id: String, val name: String)
```

Comment:
this looks like an API DTO rather than an app model. move it to the resource that owns it please.